src/init/wazuh/wazuh.sh

```diff
@@ -209,8 +209,8 @@ WazuhUpgrade()
     # Replace and delete ossec group along with ossec users
     OSSEC_GROUP=ossec
     if (grep "^ossec:" /etc/group > /dev/null 2>&1) || (dscl . -read /Groups/ossec > /dev/null 2>&1)  ; then
-        find $PREINSTALLEDDIR -group $OSSEC_GROUP -user root -exec chown root:wazuh {} \;
-        find $PREINSTALLEDDIR -group $OSSEC_GROUP -exec chown wazuh:wazuh {} \;
+        find $PREINSTALLEDDIR -group $OSSEC_GROUP -user root -print0 | xargs -0 root:wazuh
```

Comment:
Missing the `chown` command.
```suggestion
        find $PREINSTALLEDDIR -group $OSSEC_GROUP -user root -print0 | xargs -0 chown root:wazuh
```

Reply:
Fix it !